include/zephyr/net/net_context.h

```diff
@@ -418,7 +418,11 @@ __net_socket struct net_context {
 			uint8_t ipv4_mcast_ttl; /**< IPv4 multicast TTL */
 		};
 	};
-
+	
+#if defined(CONFIG_NET_IPV4)
+    	bool ipv4_mcast_loop;
```

Comment:
No spaces before a tab

```suggestion
	bool ipv4_mcast_loop;
```